### PR TITLE
Standardize imports

### LINKED ISSUE #52: Standardize imports
**Is your feature request related to a problem? Please describe.**
Imports are a bit messy throughout the code base. We should standardize this and enforce it through Codacy or CI linting.

**Describe the solution you'd like**
Standardize towards the following:

- always use absolute imports
- go for google code order (see flake8-import-order below)
  - could be used for a checker [flake8-import-order](https://pypi.org/project/flake8-import-order/)
  - use [isort](https://pypi.org/project/isort/) to apply import order
- implement in Codacy as far as possible
- implement in CI if not possible in Codacy

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by eudesbarbosa**
closes #52 
addresses #6 

**Changes**
- Standardized imports
- Added `flake8-import-order` and `isort` import definitions to `setup.cfg`
- Fixed a couple of 'token' issues from Codacy

### CODE REVIEW

**Comment by eudesbarbosa** [comment]
@holtgrewe, quick overview:
1. I passed `isort` once, even though it has a slightly different style than `flake8`. We can solve the issues slowly as we move forward.
2. The new import should be part of the CI workflow now.
3. I also took some time to remove `token` from the code.

File: README.rst [hunk 1]
```diff
@@ -12,7 +12,7 @@
     :alt: Documentation Status
 
 ===============================================
-SNAPPY - SNAPPY Nucleic Acid Procesing Pipeline
+SNAPPY - SNAPPY Nucleic Acid Processing Pipeline
```
Comment (holtgrewe) [inline README.rst:15]: Adjust the `===`.

File: requirements/test.txt [hunk 2]
```diff
@@ -17,10 +17,13 @@ pyfakefs
 black ==20.8b1
 # "snakefmt" code formatter and checker.
 snakefmt ==0.3.1
+# isort code formatter - import order
+isort==4.3.21
```
Comment (holtgrewe) [inline requirements/test.txt:21]: We have a space before `==` elsewhere.

**Review by holtgrewe — CHANGES_REQUESTED** [review]
Looks good in general, just adress the points.

Please add isort to here:

- https://github.com/bihealth/repository-checklists/blob/main/Checklist-Python.md

File: requirements/test.txt (hunk 2 above)
Comment (eudesbarbosa) [reply requirements/test.txt]: Fixed.

**Comment by eudesbarbosa** [comment]
Added the style checking to the list.

The last Codacy issue I cannot fix for the life of me...

**Review by eudesbarbosa — COMMENTED** [review]
ok

**Comment by eudesbarbosa** [comment]
> 
> 
> Looks good in general, just adress the points.
> 
> Please add isort to here:
> 
>     * https://github.com/bihealth/repository-checklists/blob/main/Checklist-Python.md

All done. New PR: https://github.com/bihealth/repository-checklists/pull/1

**Review by holtgrewe — APPROVED** [review]

**Comment by holtgrewe** [comment]
Can be merged IMO, make sure to squash ;-) 🔨

**Pull request merged**